zstacklib/zstacklib/utils/blkdiscard.py

```
@@ -0,0 +1,40 @@
+            offset += batch
+    finally:
+        os.close(fd)
+        sys.stdout.write("finished.\n")
+
+if __name__ == "__main__":
+    if len(sys.argv) != 5:
+        sys.stderr.write("Usage: {} <device> <offset> <length> <batch_bytes>\n".format(sys.argv[0]))
+        sys.exit(1)
+
+    if sys.version_info[0] < 3:
+        to_long = long
```

Ruff (0.12.2)

Undefined name `long`

(F821)